docs_outdoornav_user_manual/release_notes.mdx

```diff
@@ -6,6 +6,36 @@ toc_min_heading_level: 2
 toc_max_heading_level: 4
 ---
 
+## 0.7.0
+
+### New Features
+
+-   Goal terminology has been removed from the mission generation nomenclature (see 
+    [Definitions](/docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode#definitions)) 
+    Users can now add tasks, apply final headings, and set navigation tolerances 
+    to any Waypoint in a mission.
+-   Drag and Drop of waypoints now available in Edit Mode.
+-   New Waypoints can be inserted between existing Waypoints in a mission. 
+-   Mission API now available to create/edit/load missions, waypoints and tasks.
+-   Mission execution via mission ID is now available.
+-   The base station location is now displayed in the UI after carrying out an automated survey.
+-   New coloring scheme for GNSS status icons to provide more accurate information.
+
+### Bug Fixes
+
+-   1069: Fixed the test script for versions pre-0.5

```

Comment:
What does the "pre-0.5" part mean? How does that apply to 0.7?

Reply:
Ah I just copied the title from the ticket. I'll change the "for versions pre-0.5" and just say something like "Fixed the test script for older versions"

Reply:
I might just skip this bug fix. I'm unclear on how fixes to the test script for 0.4 installations are part of the 0.7 release. Customers might be confused also.